pkg/centralsensor/caps_list.go

```diff
@@ -33,4 +33,7 @@ const (
 
 	// NodeScanningCap identifies the capability to scan nodes and provide node components for vulnerability analysis.
 	NodeScanningCap SensorCapability = "NodeScanning"
+
+	// ListeningEndpointsCap identifies the capability for sensor to process and send information about listening endpoints and their processes, AKA processes listening on ports
+	ListeningEndpointsCap SensorCapability = "ListeningEndpoints"
```

Comment:
Since the sensor already sends listening endpoints at the deployment level but doesn't include processes should you call this something like `ListeningEndpointsWithProcessesCap` ?

Reply:
Done

Reply:
Please change the string to ListeningEndpointsWithProcesses as well.